abtesting/ABTestingExample/Tests macOS/Tests_macOS.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Copyright 2021 Google LLC
```

Comment:
Why mac OS but not tvOS?

Reply:
For future implementation, not currently in use.